### ISSUE TITLE
Q&A: i18n Feature

### ISSUE THREAD

**Issue by luatdolphin** [amiroff/NextBook#14]
Hello. Can you help me about i18n, please? Will this project include i18n features into the source code?
Thanks!!!!!!

**Comment by amiroff** [comment]
@luatdolphin can you describe your specific needs? Do you need to create a book in multiple languages?

**Comment by luatdolphin** [comment]
Yes. I mean. I can choose multiple languages of one book from navigation menu

**Comment by amiroff** [comment]
You can actually achieve this by putting books in different folders like /pages/en/mybook and /pages/it/mybook and link to them in config.json accordingly. I would love to add an officiat implementation of this to features list though. 

Meanwhile you can try a similar project with i18n features already present: [Nextra](https://nextra.vercel.app/)

**Comment by luatdolphin** [comment]
I am still looking forward to your project and use your source. Thank for your support.
